deliveries/tdot-milestone-1.md

```diff
@@ -0,0 +1,21 @@
+# Milestone Delivery :mailbox:
+
+> ⚡ Only the GitHub account, which is responsible for the pull request of the accepted application is allowed to submit milestones.
+>
+> Don't remove any of the mandatory parts presented in bold letters or as headlines! Lines starting with `>`, such as this one, can be removed.
+
+**The [invoice form :pencil:](https://forms.gle/8Wx7nxtq8fKrsuEz8) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/General-Grants-Program/blob/master/grants/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** [tDOT](https://github.com/w3f/Grants-Program/blob/master/applications/tdot.md).
+* **Milestone Number:** 1
+
+> Please provide a list of all deliverables of the milestone extracted from the initial application and a link to the deliverable itself. Ideally all links inside the below table should include a commit hash, which will be used for testing. If you don't provide a commit hash, we will work off the default branch of your repository. Thus, if you plan on continuing work after delivery, we suggest you create a separate branch for either the delivery or your continuing work.
+>
+> If there is anything particular about any of the deliverables we or a future reader should know, use the respective `Notes` column.
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 0b. | Documentation |[architecture](https://github.com/nutsfinance/stable-asset/blob/affe16c004003307bfa9c5e3db733d78e1c9f33e/architecture.png)||
+| 0c.  | Testing |[code](https://github.com/nutsfinance/stable-asset/blob/affe16c004003307bfa9c5e3db733d78e1c9f33e/lib/stable-asset-xcm/src/tests.rs)||
```

Comment:
Could you please link specifically to the testing described in this milestone "comprehensive tests that covers stable asset pool management functionalities"?

Reply:
Updated